documentation/working-with-testcafe-studio/recording-tests/test-actions/on-page-actions/action-parameters.md

```diff
@@ -17,18 +17,18 @@ You can set up the *Element Selector* parameter in one of the following ways:
 
     ![Element selector](../../../../../images/actions/element-selector.png)
 
-    TestCafe Studio produces a set of selectors for a target element. You can view them in in the **Element Selector** drop-down list and pick the one that fits you best.
+    TestCafe Studio produces a set of selectors for a target element. These selectors are available in the **Element Selector** drop-down list.
 
     ![Selectors list](../../../../../images/actions/selectors-list.png)
 
 * **Generate a selector by picking an element on a webpage**
 
-    If you add an on-page action to a test from the *Actions* panel during recording, you can select a target element on the tested webpage using an element picker. To do this, click the ![Element Picker](../../../../../images/getting-started/element-picker-icon.png) button next to the **Element Selector** field, then pick the element on the webpage. TestCafe Studio generates the element selector and fills the **Element Selector** field with it. You can view other available selectors in the **Element Selector** drop-down list.
+   You can select a target element on the tested webpage using an element picker when you add an on-page action from the *Actions* panel. To do this, click the ![Element Picker](../../../../../images/getting-started/element-picker-icon.png) button next to the **Element Selector** field and select the element on the webpage. TestCafe Studio generates the element selector and adds it to the **Element Selector** field.
```

Comment:
@dirk-pieterse 

I think we should use my paragraph. The first sentence introduces the user to the situation when it can pick an element and generate a selector: "If you add an on-page action to a test".  
It's like in the previous paragraph. First it describes when the selector is generated automatically: "When you interact with a webpage during recording, ..."

Reply:
If... If... When... When... To... To... You can just state the situation: You can do this...; You can use this...

Reply:
Ok, one more moment - you remove "during recording". It's important. Let's rewrite in this way: "You can select a target element on the tested webpage during recording using an element picker "

Reply:
You can use an element picker to select a target element on the tested webpage during recording.